mycroft/skills/twitter/dialog/twitter.success.dialog

```diff
@@ -0,0 +1,3 @@
+I have sent your post to Twitter.
+Tweet sent!
+Okay, I am fulfilling your request.
```

Comment:
There is a copy of the .dialog files in the `en-us` folder, but there is also a copy of each of them in the root dialog folder still. That should be fixed.